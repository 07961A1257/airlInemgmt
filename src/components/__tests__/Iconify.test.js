import React from 'react';
import Iconify from '../Iconify';
import { shallow } from 'enzyme';

it('Iconify Component', () => {
  const wrapper = shallow(<Iconify />);
  expect(wrapper).toMatchSnapshot();
});
