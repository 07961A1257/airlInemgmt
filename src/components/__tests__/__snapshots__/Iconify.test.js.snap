// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Iconify Component 1`] = `ShallowWrapper {}`;
